README.rst

```diff
@@ -103,16 +103,17 @@ Mapping
 -------
 
 The MassiveSearchBundle requires that you define which objects should be indexed
-through *mapping*. Currently only **XML mapping** supported:
+through *mapping*. Currently only **XML mapping** is natively supported:
```

Comment:
XML is supported natively (i.e. it comes with XML support) - but it implies that you can add your own metdata drivers.